MOM6_GEOSPlug/mom6_app/mk_MAPL_Tripolar/mk_MAPL_Tripolar.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python 
```

Comment:
@yvikhlya Query: Is this a Python3 script? If so, you might want to change the shebang to:
```
#!/usr/bin/env python3
```

There are still systems where `python` might resolve to `python2` (like discover). I find it's always best (for now) to point to `python3` directly so you don't catch any python2 issues.

Reply:
Fixed.